### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by MatthiasWanner**
Update master with dev branch

### CODE REVIEW

File: src/components/NavBar.tsx/ADMIN.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```
Comment (MatthiasWanner) [inline src/components/NavBar.tsx/ADMIN.tsx]: Please rename folder by NavBar (delete ".tsx")

File: src/components/NavBar.tsx/ADMIN.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import Collaborateurs from '../../../media/icons/Collaborateurs.svg';
+interface ISPNavbar {
+  handleClose: () => void;
+  Home: string;
+  Rapport: string;
+  Réglages: string;
+  NouveauRapport: string;
+}
+
+function ADMIN({ Home, Rapport, Réglages, NouveauRapport, handleClose }: ISPNavbar): JSX.Element {
```
Comment (MatthiasWanner) [inline src/components/NavBar.tsx/ADMIN.tsx]: Règlages, NouveaRapport, Rapport? Please use english for your props name

File: src/components/NavBar.tsx/SUPERADMIN.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import { Link } from 'react-router-dom';
+
+interface ISPNavbar {
+  handleClose: () => void;
+  Home: string;
+  Rapport: string;
+  Réglages: string;
+}
+
+function SUPERADMIN({ Home, Rapport, Réglages, handleClose }: ISPNavbar): JSX.Element {
```
Comment (MatthiasWanner) [inline src/components/NavBar.tsx/SUPERADMIN.tsx]: Idem, english props name

File: src/components/NavBar.tsx/USER.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+interface ISPNavbar {
+  handleClose: () => void;
+  Home: string;
+  Rapport: string;
+  Réglages: string;
+  NouveauRapport: string;
+}
+
+function USER({ NouveauRapport, Home, Rapport, Réglages, handleClose }: ISPNavbar): JSX.Element {
```
Comment (MatthiasWanner) [inline src/components/NavBar.tsx/USER.tsx]: Props name in english

File: src/components/NavBar.tsx/sideBar.tsx [hunk 5]
```diff
@@ -0,0 +1,118 @@
+import React, { Dispatch, SetStateAction, useState } from 'react';
+import { connect } from 'react-redux';
+import Home from '../../../media/icons/Home.svg';
+import Rapport from '../../../media/icons/folder.svg';
+import Réglages from '../../../media/icons/Settings.svg';
```
Comment (MatthiasWanner) [inline src/components/NavBar.tsx/sideBar.tsx]: English

**Review by MatthiasWanner — COMMENTED** [review]
All is ok for me

File: src/components/NavBar.tsx/USER.tsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+interface ISPNavbar {
+  handleClose: () => void;
+  Home: string;
+  report: string;
+  settings: string;
+  newReport: string;
+}
+
+function USER({ newReport, Home, report, settings, handleClose }: ISPNavbar): JSX.Element {
```
Comment (Karnak19) [inline src/components/NavBar.tsx/USER.tsx]: Be consistent in naming convention (i.e. camelCase)

File: src/components/NavBar.tsx/sideBar.tsx [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+import newReport from '../../../media/icons/NouveauRapport.svg';
+import SUPERADMIN from './SUPERADMIN';
+import ADMIN from './ADMIN';
+import USER from './USER';
+import store, { actions, RootState } from '../../assets/redux/store';
+import { useHistory } from 'react-router-dom';
+import Togglebutton from '../../assets/ToggleButton';
+import { today } from '../../assets/date';
+
+interface sideBarProps {
```
Comment (Karnak19) [inline src/components/NavBar.tsx/sideBar.tsx]: PascalCase for interfaces

File: src/components/companies/CompanyDetails.tsx [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+  );
+
+  if (isLoading) return <p>Loading ...</p>;
+
+  if (error)
+    return (
+      <p>
+        error : {error.message} {error.code}
+      </p>
+    );
```
Comment (Karnak19) [inline src/components/companies/CompanyDetails.tsx]: Don't you have a Loader and Error component ?

File: src/components/companies/ListsCompanies.tsx [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+  const searchInput = watch('search');
+
+  if (isLoading) return <p>Loading ...</p>;
+
+  if (error)
+    return (
+      <p>
+        error : {error.message} {error.code}
+      </p>
+    );
```
Comment (Karnak19) [inline src/components/companies/ListsCompanies.tsx]: Same here, Loading & Error components ?

File: src/components/head.tsx [hunk 10]
```diff
@@ -10,19 +10,21 @@ interface IProps {
 function Head({ setIsSidebarVisible, setSideBarClass }: IProps): JSX.Element {
```
Comment (Karnak19) [inline src/components/head.tsx:10]: File naming (i.e. PascalCase)

File: tailwind.config.js [hunk 11]
```diff
@@ -1,32 +1,64 @@
...
+      // GrayVersion
+      // colors: {
+      //   whiteGray: '#DEDDDD',
+      //   mainBg: '#202225',
+      //   input: '#404040',
+      //   componentBorder: '#40434B',
+      //   customGreen: '#58926F',
+      //   customRed: '#976C6C',
+      //   component: '#37393F',
+      // },
```
Comment (Karnak19) [inline tailwind.config.js]: dead code

File: tailwind.config.js [hunk 12]
```diff
@@ -1,32 +1,64 @@
...
-  darkMode: false, // or 'media' or 'class'
+  darkMode: 'class', // or 'media' or 'class'
   theme: {
     extend: {
       fontFamily: {
         roboto: ['Roboto', 'sans-serif'],
       },
+      backgroundImage: {
+        // bgImg: "url('./media/images/bgAevisioBlue.png')",
+        // bgImg: "url('./media/images/bgAevisioGray.png')",
```
Comment (Karnak19) [inline tailwind.config.js]: dead code

**Review by Karnak19 — CHANGES_REQUESTED** [review]

**Pull request merged**